### DISCUSSION TITLE
How to set a background image on any content page?

### DISCUSSION THREAD

**Discussion by talbx** [nunocoracao/blowfish#929] [Q&A] [answered]
Hi,

I'm struggling setting background images on any content page, so that it behaves like on the homepage when you set `layout = "background"` in `params.toml`. From the examples, and code I checked in the theme, I understood that I only have to provide a `featured.*` or `background.*` image file in the directory (i.e `content/my-services/background.jpg`).

Since that does not work, I'm asking you guys, how do I achieve that?

Thanks in advance

**Comment by wolfspyre** [comment, marked as answer]
@talbx  
- look at the layouts... what **I'm** seeing is a combination of using the `heroStyle` and `showHero` params for the pagetype in question in conjunction with a featured.(imagetype) || background.(imagetype) alongside the pagebundle.. but someone more knowledgable may have a better answer?

**Reply by talbx** [reply]
@wolfspyre thank you for your answer. You were totally right. adding 
```yaml
heroStyle: "background"
showHero: true
```
to a page does indeed add the background image to it. This solves my issue. I will try to document this in the theme docs, so other people won't struggle with it like me.

**Comment by wolfspyre** [comment]
that makes me happy 🫶
Glad to hear I helped!
good call on the docs! @nunocoracao - Where in the docs do you think makes the most sense to highlight this?  Would you do it with an exemplar page? or is there someplace that you think would be more thematically apropos ?
